src/cordova/android/OutlineAndroidLib/outline/src/main/aidl/org/outline/TunnelConfig.aidl

```diff
@@ -14,10 +14,8 @@
 
 package org.outline;
 
-import org.outline.shadowsocks.ShadowsocksConfig;
-
 parcelable TunnelConfig {
   String id;
-  String name;
-  ShadowsocksConfig proxy;
+  String host;
```

Comment:
Revert. We need a service name, not a host address.
Perhaps we should rename the field to serviceName.